assets/stylesheets/sections/google-my-business.scss

```diff
@@ -1,5 +1,7 @@
 /** @format */
+@import '../shared/colors';
```

Comment:
Do we need these imports?

Reply:
We do, since we're using color variables in the stylesheet for this new `Select Location` page.